js/index.js

```diff
@@ -0,0 +1,159 @@
+
+
+
+let navBar = document.querySelector('nav');
+let firstNewTag = document.createElement('a');
+firstNewTag.setAttribute('href', '#');
+firstNewTag.innerHTML = 'Home';
+let secondNewTag = document.createElement('a');
+secondNewTag.setAttribute('href', '#');
+secondNewTag.innerHTML = 'Test';
+navBar.prepend(firstNewTag);
+navBar.appendChild(secondNewTag);
+navBar.style.color = 'green';
+
+
+let contentHeaders = document.getElementsByTagName("h4");
+contentHeaders[0].innerHTML = (siteContent["main-content"]["features-h4"])
+contentHeaders[1].innerHTML = (siteContent["main-content"]["about-h4"])
+contentHeaders[2].innerHTML = (siteContent["main-content"]["services-h4"])
+contentHeaders[3].innerHTML = (siteContent["main-content"]["product-h4"])
+contentHeaders[4].innerHTML = (siteContent["main-content"]["vision-h4"])
+contentHeaders[5].innerHTML = (siteContent["contact"]["contact-h4"])
+
+let contentBody = document.getElementsByTagName("p");
+contentBody[0].innerHTML = (siteContent["main-content"]["features-content"])
+contentBody[1].innerHTML = (siteContent["main-content"]["about-content"])
+contentBody[2].innerHTML = (siteContent["main-content"]["services-content"])
+contentBody[3].innerHTML = (siteContent["main-content"]["product-content"])
+contentBody[4].innerHTML = (siteContent["main-content"]["vision-content"])
+contentBody[5].innerHTML = (siteContent["contact"]["address"])
+contentBody[6].innerHTML = (siteContent["contact"]["phone"])
+contentBody[7].innerHTML = (siteContent["contact"]["email"])
+contentBody[8].innerHTML = (siteContent["footer"]["copyright"])
+
+//Nav text color
+navItems[0].style.color = 'green';
+navItems[1].style.color = 'green';
+navItems[2].style.color = 'green';
+navItems[3].style.color = 'green';
+navItems[4].style.color = 'green';
+navItems[5].style.color = 'green';
+navItems[6].style.color = 'green';
+navItems[7].style.color = 'green';
+
+//Pointers
+let customButton = document.querySelector("button");
+let ctaHeader = document.querySelector("h1");
+let ctaImg = document.querySelector("#cta-img");
+let navLogo = document.querySelector("img");
+let middleImg = document.getElementById("middle-img")
+let pText = document.querySelectorAll("p");
+//Listeners
+
+customButton.addEventListener('click', () => {
+  ctaHeader.style.color = "red";
+});
+
+ctaImg.addEventListener('mouseover', () => {
+  ctaImg.style.visibility = "hidden";
+});
+
+ctaImg.addEventListener('mouseout', () => {
+  ctaImg.style.visibility = "visible";
+});
+
+navLogo.addEventListener('contextmenu', () => {
+  alert('Nice try!');
+});
+
+middleImg.addEventListener('dblclick', () => {
+  prompt("What are you doing to me?", "Just checking if this works...");
+});
+
+window.addEventListener("load", function(event) {
+  alert("All resources finished loading!")
+});
+
+document.addEventListener('keydown', (event) => {
+  const keyName = event.key;
+  alert('keydown event\n\n' + 'key: ' + keyName);
```

Comment:
This is next-level cool! I'm really excited to see what you do with React in the next couple weeks :grinning: